### PR TITLE
Added my name into the students section

### PR DESCRIPTION
**Pull request by marionmok**
I also changed my friend's name cause she asked me to

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://marionmok.github.io/gci15.fossasia.org

File: _data/students.yml [hunk 1]
```diff
@@ -546,7 +546,7 @@
   twitter: Pro_MYScrub
   bio: Can you differentiate the useful and the useless?
 
-- name: Yee Jia Shuen
+- name: Yee J. S.
```
Comment (jigyasa-grover) [inline _data/students.yml]: Why was this changed @marionmok ?

Comment (mananwason) [inline _data/students.yml]: @marionmok Why have you changed this? Please don't change other students info. Thanks!

Comment (marionmok) [inline _data/students.yml]: She asked me to change it, but if you need I could just tell her to change it herself

Comment (jigyasa-grover) [inline _data/students.yml]: I guess, if need be, she shall change her name on her own :)
For the time being, kindly just add up your details so that your name is there on the website too !
